test/common/run-tests

```diff
@@ -156,6 +156,28 @@ def build_command(filename, test, opts):
     cmd.append(test)
     return cmd
 
+class GlobalMachine:
+    def __init__(self):
+        self.image = testvm.DEFAULT_IMAGE
+        self.network = testvm.VirtNetwork(image=self.image)
+        self.networking = self.network.host(restrict=True)
```

Comment:
That should be controlled by `--enable-network`, i.e. `restrict=not opts.enable_network`

Reply:
Ah, I see. Fixed.